### PR TITLE
Update frontend dependencies

### LINKED ISSUE #340: Frontend dependency updates
Addressing 1 dependabot alert related to JS dependencies:

- ~~https://github.com/csg-org/CompactConnect/security/dependabot/55~~ _(withdrawn by dependabot)_
- https://github.com/csg-org/CompactConnect/security/dependabot/56
    - _Currently awaiting to transitive dependency update(s) from:_
        - `@vue/cli-plugin-eslint@~5.0.8`

These are all non-runtime dependencies related to the local webpack dev server and build tooling.

### PR DESCRIPTION
**Pull request by jsandoval81**
### Requirements List
- `yarn install --ignore-engines`

### Description List
- Updated `cross-spawn`
    - https://github.com/csg-org/CompactConnect/security/dependabot/56
    - `cross-spawn@7.x` & `cross-spawn@6.x` were able to be updated normally
    - `cross-spawn@5.x` required a new resolution + confirmation that the update to `6.x` for that case was not breaking
        - In addition to smoke testing, looking at the release notes for the parent dependency `execa` as it moved `cross-spawn` from v5 to v6 as part of the `execa@1.0.0` release, the [code & notes](https://github.com/sindresorhus/execa/releases/tag/v1.0.0) for the release suggest it was just a version bump to mark the release as stable and that there were no breaking changes.

### Testing List
- `yarn test:unit:all` should run without errors or warnings
- `yarn serve` should run without errors or warnings
- `yarn build` should run without errors or warnings
- Code review

Closes #340

### CODE REVIEW

File: webroot/yarn.lock [hunk 1]
```diff
@@ -5053,9 +5053,9 @@ cross-spawn@^6.0.0:
     which "^1.2.9"
 
 cross-spawn@^7.0.0, cross-spawn@^7.0.2, cross-spawn@^7.0.3:
-  version "7.0.3"
-  resolved "https://registry.yarnpkg.com/cross-spawn/-/cross-spawn-7.0.3.tgz#f73a85b9d5d41d045551c177e2882d4ac85728a6"
-  integrity sha512-iRDPJKUPVEND7dHPO8rkbOnPpyDygcDFtWjpeWNCgy8WP2rXcxXL8TskReQl6OrB2G7+UJrags1q15Fudc7G6w==
+  version "7.0.6"
```
Comment (ChiefStief) [inline webroot/yarn.lock:5056]: Why is this not accompanied by a resolution in the package.json file?

**Comment by ChiefStief** [comment]
Heads up that this is also in the cloudfront-csp lambda and owasp-zap authenticator lambda

**Comment by jsandoval81** [comment]
@ChiefStief It looks like yesterday dependabot withdrew the original alert and issued a revised alert. The revised alert bumped down the major version requirement to v6. This made it so that the v5 dependency for `@vue/cli-plugin-eslint` could be `resolution`ed to v6 without breaking.

I've made the corresponding updates and also updated the PR and ticket notes.

**Review by ChiefStief — APPROVED** [review]

**Comment by jsandoval81** [comment]
@jlkravitz This is ready for your review.

**Review by jlkravitz — APPROVED** [review]
@isabeleliassen Good to merge once #352 is complete.

**Review by isabeleliassen — APPROVED** [review]

**Pull request merged**